task_12/index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <title>Title</title>
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu:300,300i,400,400i,500,500i,700,700i" rel="stylesheet">
+    <link rel="stylesheet" href="dist/css/style.css">
+    <script src="https://use.fontawesome.com/4a582f4e95.js"></script>
+    <script src="src/js/time.js"></script>
+</head>
+<body>
+<div class="wrapper">
+    <div class="content">
+        <header>
+            <div class="timeBlock">
+                <i class="fa fa-clock-o" aria-hidden="true"></i>
+                <time class="time">12:55:34</time>
```

Comment:
Do you really need this class? It's the same as type.

Reply:
I think yes
I had an idea how to use it further